### PR TITLE
add a reader that can iterate an eBird CSV

### PR DESCRIPTION
**Pull request by nvcexploder**

### CODE REVIEW

**Comment by RichardLitt** [comment]
I don't think this is ready to merge? Does this include the chunked file logic?

**Comment by nvcexploder** [comment]
> I don't think this is ready to merge? Does this include the chunked file logic?

Not yet - let me add those in. I'll rename them so they're a little easier to identify, also.

File: src/ebird-ext/readSplitFile.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    'hasMedia',
+    'approved',
+    'reviewed',
+    'reason',
+    'comments',
+    'speciesComments'
+  ]
+})
+
+const filepath = '/Users/benacker/src/birdinginvermont.com/private-data/nosingles.txt'
```
Comment (RichardLitt) [inline src/ebird-ext/readSplitFile.js:70]: Should we perhaps include this as an argument from the CLI?

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: src/ebird-ext/readSplitFile.js (hunk 1 above)
Comment (nvcexploder) [reply src/ebird-ext/readSplitFile.js:70]: I think that to move it into the CLI would need some refactoring, which I can help with soon (specifically within 3-4 days). I'm 100% okay to do the refactor before merging, too. At least this gets you the functions that chunk the files for a faster dev cycle, and I can perform the refactor and documentation before an actual merge.

Comment (RichardLitt) [reply src/ebird-ext/readSplitFile.js:70]: Sweet. I'll wait up. Wanting to use the chunking to do some cool stuff with towns for a blog for https://vtecostudies.org/. Let me know when you're able to hack at it! :)

**Comment by RichardLitt** [comment]
Merged!

**Pull request closed without merging**